src/main/kotlin/net/atos/zac/mail/model/MailAdres.kt

```diff
@@ -7,11 +7,16 @@ package net.atos.zac.mail.model
 import jakarta.json.bind.annotation.JsonbProperty
 import jakarta.mail.Address
 import jakarta.mail.internet.InternetAddress
+import net.atos.zac.util.ValidationUtil
 
 class MailAdres(
     @field:JsonbProperty("Email") var email: String,
     @field:JsonbProperty("Name") var name: String?
 ) {
+    init {
```

Comment:
The @Valid annotation does not work here?

Reply:
We construct this in converters so no jsonb validation takes place
Additionally `@Email` is not very strict and allows emails like `a@b`, while we want `a@b.domain`